src/teehr/loading/nwm/retrospective_grids.py

```diff
@@ -1,4 +1,25 @@
-"""A module for loading retrospective NWM gridded data."""
+"""A module for loading retrospective NWM gridded data.
+
+The function ``nwm_retro_grids_to_parquet()`` can be used to fetch and format
+two different versions of retrospective NWM data (v2.1 and v3.0) for
+specified variables and date ranges, and summarize the grid pixels
+intersecting polygons provided in the weights file using an area-weighted mean
+approach.
+
+Each version of the NWM data is hosted on `AWS S3
+<https://registry.opendata.aws/nwm-archive/>`__ and is stored in Zarr format
+(v3.0) or as Kerchunk reference files (v2.1). Several options are included
+for fetching the data in chunks, including by week or month which can be
+specified using the ``chunk_by`` argument.
```

Comment:
@samlamont can we include the chunk sizes here?  It is pretty technical, but might be useful.

Reply:
how about something like: 
```
The NWM v3.0 retrospective forcing Zarr store has dimensions {time: 385704,
x: 3840, and y: 4608}, with a chunking scheme of {time: 672, x: 350, y: 350}.
Only the data variable chunks that intersect the polygons are read into memory.

The NWM v2.1 retrospective forcing data has the same x-y dimensions but is
fetched using Kerchunk reference files, which point to the original hourly
netcdf files, therefore the data is not chunked in the x-y dimensions. This
means that an entire data variable is read into memory regardless of the
spatial bounds of the polygons being processed.
```